web/react/utils/markdown.jsx

```diff
@@ -69,6 +69,15 @@ class MattermostMarkdownRenderer extends marked.Renderer {
             usedLanguage = 'xml';
         }
 
+        if (usedLanguage && (usedLanguage === 'tex' || usedLanguage === 'tex')) {
```

Comment:
I imagine this is a mistake :P `usedLanguage === 'tex' || usedLanguage === 'tex'`